packages/react-icons/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@coveord/plasma-react-icons",
-    "version": "46.0.2-next.3",
+    "version": "46.0.2",
```

Comment:
im not good at math but this does not read 47 🙈

Reply:
The commit is tagged as breaking, so it will release 47.0.0 and bump everywhere